cmake/RunTests.cmake

```diff
@@ -38,7 +41,7 @@ if(DEFINED ENV{TEST_FILTER} AND NOT "$ENV{TEST_FILTER}" STREQUAL "")
 endif()
 
 # TMPDIR: use relative test path (for parallel test runs / isolation).
-set(ENV{TMPDIR} "${BUILD_DIR}/Xtest_tmpdir/${rel_test_path}")
+set(ENV{TMPDIR} "${BUILD_DIR}/Xtest_tmpdir/${TEST_PATH}")
```

Comment:
We don't need `rel_test_path` , instead I changed it to always force `TEST_PATH` to a relative path. (The code simplification is worth the extra subdirectory beneath `Xtest_tmpdir/`)